contrib/direnv

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
+
+# Discover PRJ_ROOT if not found via git (i.e. not in a git repository).
+# Look for the .config folder in the current directory and up
+find_prj_config() (
+	local old_pwd
+	while [[ $old_pwd != $PWD ]]; do
+		if [[ -d .config && "$HOME" != "$PWD" ]]; then
+			echo "$PWD"
+			return 0
+		fi
+		old_pwd=$PWD
+		cd ..
+	done
+	# We're at the top and didn't find anything
+	log_error "ERROR: could not find project root" >&2
+	return 1
+)
+
+# Discover PRJ_ROOT if in a git repository.
+find_prj_root_with_git (
+	# remove everything after .git in the path because worktrees
+	# have the following path: /path/to/main/.git/worktrees/<name>
+	echo "${GIT_DIR%/.git*}"
```

Comment:
Just in case the comment is missed in a resolved thread, I think this might still be a problem. See https://github.com/numtide/prj-spec/pull/4#discussion_r1192970170

Reply:
Yes indeed a problem remains. Ack!

Reply:
440705abeb8de9e63c703724a21802c89f6600c7